core/bin/dust_api.rs

```diff
@@ -1920,7 +1921,7 @@ async fn databases_tables_list(
 
 #[derive(serde::Deserialize)]
 struct DatabasesRowsUpsertPayload {
-    contents: HashMap<String, serde_json::Value>,
+    contents: Vec<DatabaseRow>,
```

Comment:
Maybe we need a third type that excludes `created` ? or just get rid of `created` entirely ?